pkg/action/cl001/ac000/plan.go

```diff
@@ -16,17 +16,17 @@ var Plan = []plan.Step{
 	},
 	{
 		Action:  "ac002",
-		Backoff: plan.NewBackoff(24*time.Minute, 3*time.Minute),
+		Backoff: plan.NewBackoff(30*time.Minute, 3*time.Minute),
 		Comment: "check cluster access",
 	},
 	{
 		Action:  "ac003",
-		Backoff: plan.NewBackoff(10*time.Second, 2*time.Second),
+		Backoff: plan.NewBackoff(30*time.Minute, 1*time.Minute),
 		Comment: "check master count",
 	},
 	{
 		Action:  "ac004",
-		Backoff: plan.NewBackoff(10*time.Second, 2*time.Second),
+		Backoff: plan.NewBackoff(15*time.Minute, 1*time.Minute),
```

Comment:
I would also not agree with this one because only creating a control plane does not create workers so if you wait 10 seconds or 15 minutes does not matter. Simply because the worker count stays zero at this step.

Reply:
Make sense I reverted my change.